Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -395,7 +396,8 @@ extension ProjectDescription.Product {
             return product
         } else if hasLibraryProducts {
             // only automatic products, default to static framework
-            return .staticFramework
+            // Products with resources should default to a dynamic framework
```

Comment:
I would replace the two comments with a single one:
`// only automatic products, default to dynamic framework if there are resources, or static framework otherwise`